client/components/GenericTable/GenericTable.tsx

```diff
@@ -57,41 +56,29 @@ const GenericTable: {
 		total,
 		pagination = true,
 		...props
-	},
-	ref,
+	}: GenericTableProps<FilterProps, ResultProps>,
+	ref: Ref<HTMLElement>,
 ) {
 	const t = useTranslation();
 
 	const [filter, setFilter] = useState(paramsDefault);
```

Comment:
From where we receive paramsDefault ?

Reply:
It doesn't matter actually, since this is the old GenericTable I kept the definitions and behaviors even if makes no sense